Wikipedia/Code/WMFWelcomeAnimationBackgroundView.swift

```diff
@@ -16,10 +16,13 @@ public struct ImageViewAndModel {
 
 open class WMFWelcomeAnimationBackgroundView: WMFWelcomeAnimationView {
 
-    open lazy var imageModels:[ImageModel]? = nil
+    func getImageModels() -> [ImageModel]? {
+        assertionFailure("Should not be called directly. Expected to be overriden by subclass")
```

Comment:
This might be clearer as a `setupImageModels` that's called once where subclassers set the `imageModels` property